server.js

```diff
@@ -1,20 +1,20 @@
-const express = require('express');
-const webpack = require('webpack');
-const webpackDevMiddleware = require('webpack-dev-middleware');
+const express = require('express')
```

Comment:
- [ ] [optional] since this is a front end project, it was unnecessary to create a nodejs server, webpack only could have worked.